README.md

```diff
@@ -11,37 +29,147 @@
+
+- **Install all configurations and tools:**
+
+  ```bash
+  sudo make install
+  ```
+
+- **Install only system configurations:**
+
+  ```bash
+  sudo make install-etc
+  ```
+
+- **Install tools and applications:**
+
+  ```bash
+  sudo make install-tools
+  ```
+
+- **Run tests to validate configurations:**
+
+  ```bash
+  make test
+  ```
 
-Want to test some stuff? Then use this:
+- **Clean up temporary files:**
+
+  ```bash
+  make clean
+  ```
+
+For a full list of available commands, refer to the `Makefile`.
+
+## Directory Structure
+
+- **payload/**
+  - **bin/**: Custom bash scripts and tools.
+  - **config/**: Configuration files for various applications and services.
+  - **Setup/**: Setup scripts for installing and configuring software.
+  - **etc/**: System configuration files (e.g., `bashrc`, `environment`).
+- **tests/**: Bash scripts for testing configurations and setups.
+- **Makefile**: Automates installation, testing, and cleanup tasks.
+- **dot_files/**: Template dotfiles like `.bashrc`, `.profile`.
+
+## Scripts Overview
+
+### Payload Bin Scripts
+
+Located in **`payload/bin/`**, these scripts perform various system tasks:
+
+- **`ssl_banner_sniff.bash`**: Extracts SSL certificates from a remote host.
+- **`ssl_banner_grab.bash`**: Grabs SSL banners for analysis.
+- **`makedict.sh`**: Processes text files to generate sorted word lists.
+- **`getAppID.bash`**: Retrieves the application identifier of a macOS app.
+- **`entropy_feed.bash`**: Generates entropy to keep SSH connections alive.
+- **`printBanner.sh`**: Prints text in banner form.
+- **`openssl_x509_text.sh`**: Extracts X.509 certificate details.
+
+### Setup Scripts
+
+Found in **`payload/Setup/`**, these scripts automate software installation:
+
+- **`install_homebrew.bash`**: Installs Homebrew and essential packages.
+- **`install_zoom.bash`**: Installs Zoom client.
+- **`install_discord.bash`**: Installs Discord client.
+- **`install_gpg_tools.bash`**: Installs GPG tools for encryption.
+- **`install_sonic_visualiser.bash`**: Installs Sonic Visualiser for audio analysis.
+- **`install_macs_fan_control.bash`**: Installs Macs Fan Control for temperature monitoring.
+- **`install_steam.bash`**: Installs Steam gaming platform.
+- **`install_signal.bash`**: Installs Signal private messenger.
+- **`install_brew_bundle.bash`**: Installs packages from a Brewfile.
+
+### Test Scripts
+
+Located in **`tests/`**, these scripts validate configurations:
+
+- **`test_spell_lintian.sh`**: Checks for spelling errors in scripts.
+- **`test_git_config.sh`**: Validates Git configuration files.
+- **`test_pf_config.sh`**: Validates PF firewall configuration files.
+- **`test_plist.sh`**: Validates plist files for proper XML formatting.
+- **`test_bash_env.sh`**: Tests the bash environment scripts.
+- **`test_sh_lock.sh`**: Tests custom `shlock` implementation.
+- **`test_codecov_yaml.sh`**: Validates the `codecov.yml` configuration.
+
```

Comment:
🙅🏻 They are all `-` in the markdown source, which is consistent. I believe you are confusing a rendered copy of the GH markdown, or otherwise mistaken. The `-`, in the `README.md` file, are already consistent and would need to all stay or all change together, not selectively; your suggestion is divergent not consistent.